src/detect.h

```diff
@@ -1137,9 +1134,6 @@ typedef struct DetectEngineThreadCtx_ {
      * prototype held by DetectEngineCtx. */
     SpmThreadCtx *spm_thread_ctx;
 
-    /** ip only rules ctx */
-    DetectEngineIPOnlyThreadCtx io_ctx;
```

Comment:
nice cleanup

Reply:
Was sort of an accident :-)  Once I noticed I could remove io_tctx->sig_match_array and use a local variable instead, I noticed there were no other references to the tctx.  I wasn't sure if I could actually remove it until I tried it and everything still worked.